airflow/www/templates/airflow/dags.html

```diff
@@ -215,13 +215,13 @@ <h2>{{ page_title }}</h2>
                   title="Status of all previous DAG runs.">info</span>
           </th>
           <th>Schedule</th>
-          <th style="width:180px;">{{ sortable_column("Last Run", "last_dagrun") }}
+          <th style="width:180px;">{{ sortable_column("Last Data Interval", "last_dagrun") }}
```

Comment:
If we want to be pedantic here, isn't this the logical date?

Reply:
Yes it is. Incorporating the next comment in a commit